### PR TITLE
Add Ember Inspector video

### PR DESCRIPTION
**Pull request by GavinJoyce**
This adds the [Ember Inspector overview video](https://www.youtube.com/watch?v=6Yl1j2thKac)  to the guides. 

screen shot 2016-09-30 at 09 08 09

### CODE REVIEW

File: source/stylesheets/components/_article.scss [hunk 1]
```diff
@@ -43,6 +43,11 @@ article {
     @extend %default-ol;
   }
 
+  .video {
```
Comment (locks) [inline source/stylesheets/components/_article.scss]: bit more margin-top maybe (or equivalent)

Comment (GavinJoyce) [inline source/stylesheets/components/_article.scss]: 👍 I added `margin-top: 2.5em`:

screen shot 2016-09-29 at 17 11 19

File: source/localizable/ember-inspector/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The Ember Inspector is a browser add-on designed to help you understand and debug your Ember applications. You can install it on [Google Chrome](installation/#toc_google-chrome), [Firefox](installation/#toc_firefox) and [other browsers](http://localhost:4567/ember-inspector/installation/#toc_via-bookmarklet) (via a bookmarklet)
+
+Here's a brief video showcasing some of the features of the Inspector:
+
+<div class="video">
+  <iframe width="640" height="360" src="https://www.youtube.com/embed/6Yl1j2thKac?feature=player_detailpage" frameborder="0" allowfullscreen></iframe>
```
Comment (acorncom) [inline source/localizable/ember-inspector/index.md]: By specifying width/height here do we cause problems on mobile pages?

Comment (GavinJoyce) [inline source/localizable/ember-inspector/index.md]: Thanks, I've updated it to be responsive using [this technique](https://www.h3xed.com/web-development/how-to-make-a-responsive-100-width-youtube-iframe-embed):

screen shot 2016-09-30 at 09 08 09

screen shot 2016-09-30 at 09 08 27

**Comment by GavinJoyce** [comment]
I've updated this to use the video on the offical youtube channel:

https://www.youtube.com/watch?v=9TMaFhYwC6g

**Comment by acorncom** [comment]
Thanks @GavinJoyce, looks great

**Comment by acorncom** [comment]
@homu r+

**Pull request merged**